prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -177,15 +177,15 @@ class SettingsFacade(
             )
 
             // Now insert
-            val accessResult = when {
+            val (accessResult, resultMetadata) = when {
```

Comment:
Why isn't the accessResult just part of the resultMetadata?

Reply:
No. Access result is an enum success or created, while resultMetadata contains version.